apps/beats-time/app.c

```diff
@@ -3,7 +3,7 @@
 #include <math.h>
 #include "watch.h"
 
-const uint8_t UTC_OFFSET = 4; // set to your current UTC offset to see correct beats time
+const int8_t UTC_OFFSET = 4; // set to your current UTC offset to see correct beats time
```

Comment:
Honestly, I saw this change but didn't review it because the standalone beats time "app" is sort of deprecated now that it's implemented as a watch face in Movement; I plan to remove this folder from apps in the near future.